src/commands/synthetics/reporters/default.ts

```diff
@@ -233,73 +236,83 @@ const getTestResultColor = (success: boolean, isNonBlocking: boolean) => {
   return chalk.bold.red
 }
 
-export const renderResults = (test: Test, results: PollResult[], baseUrl: string, locationNames: LocationsMapping) => {
-  const success = hasTestSucceeded(results)
-  const isNonBlocking = test.options.ci?.executionRule === ExecutionRule.NON_BLOCKING
-
-  const icon = renderResultIcon(success, isNonBlocking)
-
-  const idDisplay = `[${chalk.bold.dim(test.public_id)}]`
-  const nameColor = getTestResultColor(success, isNonBlocking)
-  const nonBlockingText = !success && isNonBlocking ? '[NON-BLOCKING]' : ''
-
-  const testResultsText = results
-    .map((r) => renderExecutionResult(test, r, baseUrl, locationNames))
-    .join('\n\n')
-    .concat('\n\n')
-
-  return [`${icon} ${idDisplay}${nonBlockingText} | ${nameColor(test.name)}`, testResultsText].join('\n')
-}
-
-// Other rendering
-export const renderTrigger = (test: Test, testId: string, executionRule: ExecutionRule, config: ConfigOverride) => {
-  const idDisplay = `[${chalk.bold.dim(testId)}]`
-
-  const getMessage = () => {
-    if (executionRule === ExecutionRule.SKIPPED) {
-      // Test is either skipped from datadog-ci config or from test config
-      const isSkippedByCIConfig = config.executionRule === ExecutionRule.SKIPPED
-      if (isSkippedByCIConfig) {
-        return `>> Skipped test "${chalk.yellow.dim(test.name)}"`
-      } else {
-        return `>> Skipped test "${chalk.yellow.dim(test.name)}" because of execution rule configuration in Datadog`
-      }
+export class DefaultReporter implements Reporter {
+  private write: Writable['write']
+  constructor(command: RunTestCommand) {
+    this.write = command.context.stdout.write.bind(command.context.stdout)
+  }
+  public error(error: string) {
```

Comment:
Nit: could you add blank lines before each function definition?

Reply:
this should be a lint rule.

Reply:
added the new rule and fixed it.